src/content/en/updates/2017/04/chrome-59-deprecations.md

```diff
@@ -0,0 +1,134 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the deprecations and removals in Chrome 58 to help you plan.
+
+{# wf_updated_on: 2017-05-01 #}
+{# wf_published_on: 2017-05-01 #}
+{# wf_tags: deprecations,removals,chrome59 #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome 58 to help you plan. #}
+
+# Deprecations and Removals in Chrome 59 {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In nearly every version of Chrome, we see a significant number of updates and
+improvements to the product, its performance, and also capabilities of the Web
+Platform. This article describes the deprecations and removals in Chrome 59,
+which is in beta as of April 27. This list is subject to change at any time.
+
+## Remove features from WebVR that are not in the revised spec
+
+The current implementation of WebVR, originally implemented in Chrome 52,
+contained several methods and properties that will not be in the final spec.
+Deprecation messages were added for these features for the
+[Origin Trial](https://github.com/jpchase/OriginTrials/blob/gh-pages/explainer.md)
+that started in Chrome 56 and are now being removed. These features include:
```

Comment:
Is the origin trial being removed? Or the features? I found this sentence hard to understand.